### PR TITLE
Update man pages for role commands

### PR DESCRIPTION
**Pull request by KevinAnsleyJohnson**
Signed-off-by: Kevin Johnson <kevin_johnson@cargill.com>

### CODE REVIEW

File: cli/man/grid-role-update.1.md [hunk 1]
```diff
@@ -58,27 +58,36 @@ OPTIONS
...
 
 `-k`, `--key`
-: Base name for private signing key file
+: Base name for private signing key file. (either a file path or the name of a
+  .priv file in $HOME/.grid/keys). This location can also be set via the 
+  GRID_DAEMON_KEY variable, and if not present may be set with CYLINDER_PATH
+  variable
 
-`--permissions`
+`-p`,`--permissions`
```
Comment (KevinAnsleyJohnson) [inline cli/man/grid-role-update.1.md:66]: of all the roles, only update had a short `-p`. considering altering the main.rs to either remove -p for consistency, or to add -p to the other usecases. suggestions?

Comment (dnewh) [reply cli/man/grid-role-update.1.md:66]: My opinion is that `-p` should be an option for all the usecases

Comment (KevinAnsleyJohnson) [reply cli/man/grid-role-update.1.md:66]: I'll make a separate PR for that then, because it goes beyond the scope of the task.

**Review by vaporos — CHANGES_REQUESTED** [review]
See comments on #736, which would apply here as well.

**Review by vaporos — APPROVED** [review]

File: cli/man/grid-role-update.1.md [hunk 2]
```diff
@@ -13,79 +13,83 @@ NAME
...
-: Description of the role
+: Description of the role.
 
 `--inherit-from`
-: List of roles to inherit permissions from
+: List of roles to inherit permissions from.
 
 `-k`, `--key`
-: Base name for private signing key file
+: base name or path to a private signing key file.
```
Comment (dplumb94) [inline cli/man/grid-role-update.1.md:61]: b should be capitalized

**Review by dplumb94 — CHANGES_REQUESTED** [review]

**Review by dplumb94 — APPROVED** [review]

File: cli/man/grid-role-create.1.md [hunk 3]
```diff
@@ -13,79 +13,82 @@ NAME
...
-: Set role as active
+: Set role as active. conflicts with `--inactive`.
 
 `-h`, `--help`
-: Prints help information
+: Prints help information.
 
 `--inactive`
-: Set role as inactive
+: Set role as inactive. conflicts with `--active`.
```
Comment (shannynalayna) [inline cli/man/grid-role-create.1.md:37]: Perhaps this period should be a comma, or capitalize the 'conflicts'. This happens with both instances of conflicts

**Review by dnewh — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**